packages/features/insights/server/events.ts

```diff
@@ -207,7 +207,39 @@ class EventsInsights {
       where: whereConditional,
     });
 
-    return csvData;
+    const csvDataWithBookerInfo = await Promise.all(
+      csvData.map(async (bookingTimeStatus) => {
+        if (!bookingTimeStatus.uid) {
+          return bookingTimeStatus;
+        }
+
+        const booking = await prisma.booking.findUnique({
+          where: { uid: bookingTimeStatus.uid },
+          select: {
+            attendees: {
+              select: {
+                name: true,
+                email: true,
+              },
+            },
+          },
+        });
+
+        if (!booking || !booking.attendees.length) {
+          return bookingTimeStatus;
+        }
+        const { attendees } = booking;
+        const booker = attendees[0];
```

Comment:
According to my testing, the first item in the array is the booker and the others are guests.

Reply:
one evidence:
<img width="539" alt="Screenshot 2024-10-23 at 18 39 31" src="https://github.com/user-attachments/assets/f6f82391-89ec-4ed3-ab39-c87407275208">